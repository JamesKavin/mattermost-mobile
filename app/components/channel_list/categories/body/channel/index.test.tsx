// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {renderWithIntlAndTheme} from '@test/intl-test-helper';

import ChannelItem from './index';

test('Category Channel List Item should match snapshot', () => {
    const {toJSON} = renderWithIntlAndTheme(
        <ChannelItem
            icon={'globe'}
            name='Channel Name'
        />,
    );

    expect(toJSON()).toMatchSnapshot();
});
