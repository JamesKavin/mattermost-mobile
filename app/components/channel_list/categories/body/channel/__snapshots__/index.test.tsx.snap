// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category Channel List Item should match snapshot 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingVertical": 4,
    }
  }
>
  <Icon
    name="globe"
    style={
      Object {
        "color": "rgba(255,255,255,0.72)",
        "fontSize": 24,
        "lineHeight": 28,
      }
    }
  />
  <Text
    style={
      Array [
        Object {
          "fontFamily": "OpenSans",
          "fontSize": 16,
          "fontWeight": "400",
          "lineHeight": 24,
        },
        Object {
          "color": "rgba(255,255,255,0.72)",
          "marginTop": 1,
          "paddingLeft": 12,
        },
        undefined,
      ]
    }
  >
    Channel Name
  </Text>
</View>
`;
